Code/GraphMol/MolStandardize/Validate.cpp

```diff
@@ -57,17 +73,19 @@ std::vector<ValidationErrorInfo> RDKitValidation::validate(
   return errors;
 }
 
-void NoAtomValidation::run(const ROMol &mol, bool,
-                           std::vector<ValidationErrorInfo> &errors) const {
+std::vector<ValidationErrorInfo>
```

Comment:
I think the change to use `validate` instead of `run` makes sense, but it needs to be documented in the "Backwards Incompatible Changes" section of `ReleaseNotes.md`